src/component/priceSummary/priceSummary.css

```diff
@@ -0,0 +1,36 @@
+/* For Price Total Aside  */
+
+.cart-price-heading {
+  padding: 1rem 0.5rem;
+  font-weight: 700;
+}
+
+.price-breakout {
+  padding: 0.5rem;
```

Comment:
instead of writing the same css property in different classes make it a separate helper class

Reply:
Done the changes